scripts/index.js

```diff
@@ -11,11 +11,22 @@ const {
   flags: { buildPath, publicPath, reactScriptsVersion, verbose },
 } = require('../utils/cliHandler');
 const { getReactScriptsVersion, isEjected } = require('../utils');
+
+const { major, minor, patch } = getReactScriptsVersion(reactScriptsVersion);
+
 const paths = isEjected ? importCwd('./config/paths') : importCwd('react-scripts/config/paths');
 const webpack = importCwd('webpack');
-const config = isEjected
-  ? importCwd('./config/webpack.config.dev')
-  : importCwd('react-scripts/config/webpack.config.dev');
+
+const config = (major >= 2 && minor >= 1 && patch >= 2) ?
+      (isEjected
+       ? importCwd('./config/webpack.config')
```

Comment:
Have you checked how the file looks once ejected? I would suppose it still exports a function that need the environment to be passed as argument, doesn't it?

Reply:
Yes, it's the same when ejected.

Reply:
Alright cool but we still need to update `isEjected` logic. I'd transform it to a function that accepts the version numbers and return a boolean.

Reply:
Could it not just check for the existence of config/paths.js instead?

Reply:
yes we'll do that.